### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 2

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#178] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152252145-77937a4b-e0bb-4117-9225-c9daab91f85f.png)

**Comment by AxnNxs** [comment]
Per il Principio Moltiplicativo, le combinazioni sono del tipo "pane - farcitura - salsa". Esistono però due tipi di farcitura, e può esservene uno solo alla volta. Pertanto la soluzione sarà l'unione tra le combinazioni in cui compare o solo un tipo di farcitura (farcitura1) o l'altro (farcitura2), poiché si tratta di panini differenti, e quindi elementi dell'insieme "panini possibili" disgiunti. Pertanto:

"pane - farcitura1 - salsa" U "pane - farcitura2 - salsa" = (5x7x4) + (5x6x4) = 140 + 120 = 260

**Comment by Exyss** [comment]
Oltre alla soluzione già proposta ne propongo una più immediata ed equivalente a livello matematico: consideriamo i due tipi di farciture come un'unica tipologia, ottenendo $5 \cdot 13 \cdot 4 = 260$
